### PR TITLE
CBSP-784 Rename download straight after it finishes

### PR DESCRIPTION
**Pull request by mcpaddy**

### CODE REVIEW

**Comment by Chippiewill** [comment]
interesting commit name..

**Comment by mcpaddy** [comment]
Yeah, I messed up the push.

**Comment by mcpaddy** [comment]
I have no idea why github thinks those commits are related to this branch.

**Comment by Chippiewill** [comment]
Because they're in the CBSP-784 branch and the last common parent commit between master and CBSP-784 is 46d4942 (prior to those commits). In theory you should be able to just rebase this branch onto master if you really want to get rid of them:

    $ git fetch origin
    $ git checkout CBSP-784
    $ git rebase origin/master
    $ git push origin -f

File: s3dl/s3dl.py [hunk 1]
```diff
@@ -98,6 +98,9 @@ def download_file(download):
                              download.download_path,
                              callback=progress.add_file(download.bucket,
                                                         download.key))
+    if os.path.isfile(download.file_path):
+            os.remove(download.file_path)
```
Comment (Chippiewill) [inline s3dl/s3dl.py]: style: over-indented

otherwise 👍

Comment (mcpaddy) [reply s3dl/s3dl.py]: Fixed

**Pull request merged**